### PR TITLE
doc: add users and cases, optimize format for README

### PR DESCRIPTION
**Pull request by TommyLemon**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

## How do you solve it?
Add company user cases

## Special notes for your reviewer, ex. impact of this fix, design document, etc:
None

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [x] N/A

Affects:
- [x] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

doc: add users and cases, optimize format for README

### CODE REVIEW

**Review by wey-gu — APPROVED** [review]

File: README-CN.md [hunk 1]
```diff
@@ -25,12 +25,15 @@
...
 </p>
 
-# NebulaGraph是什么？
-
+# NebulaGraph 是什么？
 
 
 **NebulaGraph** 是一款开源的图数据库，擅长处理千亿个顶点和万亿条边的超大规模数据集。
 
+已陆续被包括在 **BAT**, **TMD** 中的众多知名大厂和大量国内外中小企业使用，<br />
```
Comment (wey-gu) [inline README-CN.md:33]: Would suggest not put BAT, TMD, as it's not ture for now.

Comment (Sophie-Xie) [reply README-CN.md:33]: I think so~~pls update it, thanks.

Comment (TommyLemon) [reply README-CN.md:33]: ”在 **BAT**, **TMD** 中“
BAT contains Tencent and TMD contains Meituan

File: README.md [hunk 2]
```diff
@@ -21,7 +21,14 @@
 
 # What is NebulaGraph?
 
-**NebulaGraph** is an open-source graph database capable of hosting super large scale graphs with dozens of billions of vertices (nodes) and trillions of edges, with milliseconds of latency.
+**NebulaGraph** is an open-source graph database capable of hosting super large scale graphs <br/>
+with dozens of billions of vertices (nodes) and trillions of edges, with milliseconds of latency. 
+
+
+It has a lot of users, including some famous and big companies of **BAT**, **TMD**,  <br/>
```
Comment (wey-gu) [inline README.md:28]: The term BAT and TMD isn't a thing to English users, we may reword it here.

File: README-CN.md (hunk 1 above)
Comment (randomJoe211) [reply README-CN.md:33]: 1. 建议不要用行话，直接举例为佳。
2. 中文两个连续的并列名词之间应用顿号。
3. 有若干语法问题。

建议的修改：
包括美团、腾讯、vivo等知名大厂在内的众多企业已将 NebulaGraph 广泛应用于社交媒体、实时推荐、网络安全、金融风控、知识图谱、人工智能等领域。

Comment (TommyLemon) [reply README-CN.md:33]: > 1. 建议不要用行话，直接举例为佳。
> 2. 中文两个连续的并列名词之间应用顿号。
> 3. 有若干语法问题。
> 
> 建议的修改： 包括美团、腾讯、vivo等知名大厂在内的[众多企业](https://nebula-graph.com.cn/cases)已将 NebulaGraph 广泛应用于社交媒体、实时推荐、网络安全、金融风控、知识图谱、人工智能等领域。

之前就是这样的，但考虑到直接用腾讯、美团可能有商标侵权风险，所以才用了 BAT, TMD 这种比较宽泛的概念，而是也是说 "在 BAT, TMD 中"，并不是说这里面的公司都用了
https://github.com/vesoft-inc/nebula/pull/4572

其它的我再优化下

File: README.md (hunk 2 above)
Comment (TommyLemon) [reply README.md:28]: Any better words? We cannot directly use 'Tencent', 'Meituan' on GitHub yet😂

File: README-CN.md (hunk 1 above)
Comment (TommyLemon) [reply README-CN.md:33]: image

**Review by Sophie-Xie — APPROVED** [review]

**Review by wey-gu — APPROVED** [review]

**Pull request merged**